HIP/hip-866.md

```diff
@@ -0,0 +1,149 @@
+interactions that are desirable to Hedera.
+
+Second, we need to support the new opcodes but ensure they operate as they would
+in the absence of any Blobs.
+
+## User stories
+
+* As a smart contract deployer, I want my smart contract that expects blob
+  support to not fail in unexpected ways.
+* As a smart contract user, I will not be able to successfully use contracts
+  that depend on the presence of blobs.
+* As a future Hedera Core developer, I want the option to provide blob like
+  behavior using EIP-4844 semantics and existing opcodes.
+
+## Specification
+
+### EVM Implementation
+
+Two opcodes need to be supported. The `VERSIONEDHASH` opcode defined
+in EIP-4844 and the `BLOBBASEFEE`
+operation in [EIP-7516](https://eips.ethereum.org/EIPS/eip-7516). These opcodes
+should behave in the same way as though there are no blobs and that there never
+have been blobs.
+
+For `VERSIONEDHASH` this will result in returning all zeros whenever called, as
+there are no versioned hashes in the current transaction. This is the behavior
+if called from a legacy, Type 1, or Type 2 operation.
+
+For `BLOBBASEFEE` this will result in returning `1` at all times, as the blob
+gas cost cannot adjust below 1. This will require updating the `TxValues` object
+with a blob fee of `1`.
+
+Proper handling of the `TxValues` object in EVM transaction execution should
+allow us to use the Besu opcodes without modification. This would consist of
+always setting the `versionedHashes` to an empty list and setting
+the `blobGasPrice` to 1 Gwei.
+
+### Hedera Services Implementation
+
+The `EthereumTransaction` transaction type in Hedera will need to reject all
+type 3 transactions as invalid. The services should not need to parse the
+transactions at this time as all Type 3 transactions will be rejected.
+
+This behavior should be automatic in current implementation, as all unrecognized
+types are rejected. To verify this system tests should be updated to add a valid
+Type 3 transaction and an invalid transaction starting with the bytes `0x03`.
+
+### JSON-RPC Relay Implementation
+
+The JSON-RPC relay will need to be updated to reject type 3 transactions. There
+are two main locations this could be seen. The `eth_sendRawTransaction` call may
+receive fully formed Type 3 transactions. Those should be detected and rejected
+with a specific error message. We could parse the transaction and ensure it is a
+well-formed Type 3 transaction and provide different rejection messages. For now
+this will be a distinction without a difference as both valid and invalid Type 3
+transactions will be rejected.
+
+Second, we will begin to see calls to the simulation and estimation
+APIs (`eth_estimateGas`, `eth_call`) that may include fields indicating it is a
+blob transaction. In those cases we should similarly detect that a blob
+transaction is being simulated and reject the transaction before sending it to
+the simulation and estimation engines.
+
```

Comment:
Q: Is it possibel for the mirror node [api/v1/contracts/call](https://testnet.mirrornode.hedera.com/api/v1/docs/#/contracts/contractsCall) to determine a type 3 transaction based on input params - I don't think so but I'm not sure.

If it is able to determine then you should add a `### Mirror Node web3 Module Implementation` that  will reject type 3 transactions

Reply:
Yes.  The first byte of the Ethereum Transaction is `0x03`.  Section added.